hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -51,6 +51,16 @@
 
   private HashMap<String, NodeConfig> nodeConfigMap = new HashMap<>();
 
+  public HdfsFrameworkConfig() {
+    // The path is configurable via the mesos.conf.path system property
+    // so it can be changed when starting up the scheduler via bash
+    Properties props = System.getProperties();
+    Path configPath = new Path(props.getProperty("mesos.conf.path", "etc/hadoop/mesos-site.xml"));
+    Configuration configuration = new Configuration();
+    configuration.addResource(configPath);
+    setConf(configuration);
```

Comment:
I would love to make this via a factory and injectable for testing.. however.  I have a lot of changes I want to make here... lets wait for the next PR.